data_structures/arrays/longest_consecutive_sequence.py

```diff
@@ -0,0 +1,65 @@
+"""
+author: Sarthak Sharma https://github.com/Sarthak950  https://sarthak950.netlify.app
+date:   4 OCT 2023
+Longest Consecutive Sequence Problem from LeetCode
+
+"""
+
+
+class Solution:
```

Comment:
```suggestion

```
This doesn't need to be in a class.

Reply:
I did this thing as to be a perfect solution for leet code should i fix that